src/pages/Merchant/LoginPage/LoginPage.js

```diff
@@ -53,11 +53,13 @@ function LoginPage() {
 
   const handleMerchantAuth = (merchantObj) => {
     dispatch({
-      type: "IS_MERCHANT_LOGGEDIN",
+      type: "IS_MERCHANT_LOGGED",
```

Comment:
dont use IS_MERCHANT_LOGGED directly instead make a actions file with all the actionTypes named like export const type1="TYPE_1"

and also while login or signup same thing

This is because we need this string at 2 places so preventing any typo by using this method

Reply:
sale k maarunga b...........mne puchi to thi ki alg action file bnalu k jb to nu bola ki jrurt na h